CHANGELOG.md

```diff
@@ -1,6 +1,10 @@
 # CHANGELOG
 Inspired from [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 
+## [2.2.1]
```

Comment:
I believe 2.1.1 is mislabeled and should have been 2.2.0? @harshavamsi

This should say "unreleased"?

Reply:
Changed

------

I know you didn't ask me, but looking at the release notes on GitHub, it looks like the release notes for both versions go merged. I think everything but the notes for the SigV4 fixes should be moved to a new block for 2.2.0. I can do that if you would like.

Reply:
Please! We made a mess.

Reply:
Latest commit should have resolved this. Let me know if you want anything else done or changed.